internal/handler/handler_test.go

```diff
@@ -230,3 +236,607 @@ func TestNewHtmlPdfService(t *testing.T) {
+				if err != nil {
+					t.Errorf(err.Error())
+				}
+				diff := testutil.Diff(r, respModel.Response{
+					Status:  http.StatusOK,
+					Message: "SUCCESS",
+					Data: map[string]interface{}{
+						"id": "1",
+					},
+				})
+				if diff != "" {
+					t.Error(testutil.Callers(), diff)
+				}
+				diff = testutil.Diff(err, nil)
+				if diff != "" {
+					t.Error(testutil.Callers(), diff)
+				}
+			},
+		},
+		{
+			name:        "Failure:: Replace:: id not found",
+			requestBody: "1",
+			setupFunc: func() (*http.Request, *htmlPdfService) {
+				b := new(bytes.Buffer)
+				mockLogicier := mock.NewMockHtmlPdfServiceLogicIer(mockCtrl)
+				rec := &htmlPdfService{
+					logic: mockLogicier,
+				}
+				return httptest.NewRequest(http.MethodPut, "/v1/register/", b), rec
+			},
+			validateFunc: func(x *httptest.ResponseRecorder) {
+				if x.Code != http.StatusBadRequest {
+					t.Errorf("want %v got %v", http.StatusBadRequest, x.Code)
+				}
+				var r respModel.Response
+				got, err := io.ReadAll(x.Body)
+				if err != nil {
+					t.Errorf(err.Error())
+				}
+				err = json.Unmarshal(got, &r)
+				if err != nil {
+					t.Errorf(err.Error())
+				}
+				diff := testutil.Diff(r, respModel.Response{
+					Status:  http.StatusBadRequest,
+					Message: codes.GetErr(codes.ErrIdNeeded),
+					Data:    nil,
+				})
+				if diff != "" {
+					t.Error(testutil.Callers(), diff)
+				}
+				diff = testutil.Diff(err, nil)
+				if diff != "" {
+					t.Error(testutil.Callers(), diff)
+				}
+			},
+		},
+		{
+			name:        "Failure:: Replace:: parse multipart failure",
+			requestBody: "1",
+			setupFunc: func() (*http.Request, *htmlPdfService) {
+				b := new(bytes.Buffer)
+				y := multipart.NewWriter(b)
+				part, _ := y.CreateFormFile("file", "some-file")
+				_, err := part.Write([]byte("abc"))
+				if err != nil {
+					t.Errorf(err.Error())
+				}
+				y.Close()
+				mockLogicier := mock.NewMockHtmlPdfServiceLogicIer(mockCtrl)
+				rec := &htmlPdfService{
+					logic: mockLogicier,
+				}
+				r := httptest.NewRequest(http.MethodPut, "/v1/register/", b)
+				r = mux.SetURLVars(r, map[string]string{"id": "1"})
+				return r, rec
+			},
+			validateFunc: func(x *httptest.ResponseRecorder) {
+				if x.Code != http.StatusBadRequest {
+					t.Errorf("want %v got %v", http.StatusBadRequest, x)
+				}
+				var r respModel.Response
+				got, err := io.ReadAll(x.Body)
+				if err != nil {
+					t.Errorf(err.Error())
+				}
+				err = json.Unmarshal(got, &r)
+				if err != nil {
+					t.Errorf(err.Error())
+				}
+				diff := testutil.Diff(r, respModel.Response{
+					Status:  http.StatusBadRequest,
+					Message: codes.GetErr(codes.ErrFileSizeExceeded),
+					Data:    nil,
+				})
+				if diff != "" {
+					t.Error(testutil.Callers(), diff)
+				}
+				diff = testutil.Diff(err, nil)
+				if diff != "" {
+					t.Error(testutil.Callers(), diff)
+				}
+			},
+		}, {
+			name:        "Failure:: Replace:: parse multipart failure",
```

Comment:
Name should be denoting Form file failure.

Reply:
done

Reply:
Looks like the this case is misplaced with the above test case.

Reply:
done